yolapy/models/user.py

```diff
@@ -0,0 +1,54 @@
+from yoconfig import get_config
+from yolapy.services import Yola
+
+
+class User(object):
+
+    """Yola User."""
+
+    def __init__(self, active=False, deleted=None, email=None, id=None,
+                 name=None, partner_id=None, preferences=None,
+                 signup_date=None, surname=None):
```

Comment:
Writing optional arguments explicitly: I hate writing code like this but I like using code like this.

When `**kwargs/setattr` is applied, the user must rely on your docstring. Docstrings too easily go stale.

When the API changes, explicitly written arguments make it much more obvious.

One should always write out the optional arguments unless the number of arguments is unmanageable. I could be persuaded that the upper-limit of manageable is ~7 and this needs to be changed.